examples/llm/src/flash_attention.py

```diff
@@ -25,16 +25,15 @@ class FlashAttention(nn.Module):
     """
 
     def __init__(self, num_heads, softmax_scale=None, device=None, dtype=None):
```

Comment:
is this entire file still needed? Or can we get away with a single import from `flash-attn` with a `triton` flag?

Looking at this module here: https://github.com/HazyResearch/flash-attention/blob/780e8eeabb84fe3f41e8244f04521743b032ba35/flash_attn/modules/mha.py#L39

Reply:
Yeah I wanted to do that, but that version does not enable bias; this means we can't use alibi and have no way of handleing key_pad_mask (when we need to handle it)